packages/firestore/test/unit/local/indexeddb_persistence.test.ts

```diff
@@ -612,23 +616,19 @@ describe('IndexedDb: allowTabSynchronization', () => {
 
   after(() => SimpleDb.delete(INDEXEDDB_TEST_DATABASE_NAME));
 
-  it('rejects access when synchronization is disabled', () => {
-    return withPersistence('clientA', async db1 => {
-      await db1.start();
-      await withPersistence('clientB', async db2 => {
-        await expect(db2.start()).to.eventually.be.rejectedWith(
-          'There is another tab open with offline persistence enabled.'
-        );
-      });
+  it('rejects access when synchronization is disabled', async () => {
+    await withPersistence('clientA', async db1 => {
```

Comment:
Nit: This `await` is inconsistent with the rest of the surrounding tests.

Reply:
The previous version had a bunch of `await`s...